### PR TITLE
deps update

### PR DESCRIPTION
**Pull request by nschloe**
Deps update and some convenience changes. You can run `gulp` twice in a row without failing now, for example. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ npm run build
 ```
 and that the selenium drivers are installed and up-to-date,
 ```
-./node_modules/gulp-protractor/node_modules/.bin/webdriver-manager update
+`npm bin`/webdriver-manager update
```
Comment (andrenarchy) [inline README.md]: with node 5.0.0 and npm 3.3.6 the file is still under `./node_modules/gulp-protractor/node_modules/.bin/webdriver-manager`. Which version do you use?

File: package.json [hunk 2]
```diff
@@ -96,7 +95,6 @@
       "exports": null,
       "depends": [
         "angular",
-        "angular-simple-logger",
```
Comment (andrenarchy) [inline package.json]: finally! :D

File: README.md (hunk 1 above)
Comment (nschloe) [inline README.md]:
```
$ node --version
v5.1.0
$ npm --version
3.3.12
$ find -name webdriver-manager
./node_modules/protractor/bin/webdriver-manager
./node_modules/.bin/webdriver-manager
$ ls -l ./node_modules/.bin/webdriver-manager
./node_modules/.bin/webdriver-manager -> ../protractor/bin/webdriver-manager
```

Comment (andrenarchy) [inline README.md]: With the same `node` and `npm` versions I get:
```
$ find -name webdriver-manager
./node_modules/gulp-protractor/node_modules/.bin/webdriver-manager
./node_modules/gulp-protractor/node_modules/protractor/bin/webdriver-manager
```

Comment (andrenarchy) [inline README.md]: Ouch, there was a stale `npm-shrinkwrap.json` file in my dir...

**Pull request merged**